inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -35,8 +35,8 @@ def application_add(args: argparse.Namespace) -> str:
 
     manifest += '<repo><repos>'
 
-    for source in args.sources:
-        manifest += '<source_pkg>' + source + '</source_pkg>'
+    source_tags = [f'<source_pkg>{source}</source_pkg>' for source in args.sources]
```

Comment:
**Suggestion:** Strip whitespace from 'source' before creating the XML tag to ensure that the XML is correctly formatted without unnecessary whitespace. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    source_tags = [f'<source_pkg>{source.strip()}</source_pkg>' for source in args.sources]
```